app/src/main/java/sdp/moneyrun/ui/game/GameLobbyActivity.java

```diff
@@ -224,6 +224,9 @@ public void onDataChange(@NonNull DataSnapshot snapshot) {
                         Intent intent = new Intent(getApplicationContext(), MapActivity.class);
                         UserDatabaseProxy pdp = new UserDatabaseProxy();
                         intent.putExtra("player", user);
+                        intent.putExtra("gameId", gameId);
+                        intent.putExtra("host", false);

```

Comment:
I think not passing anything for the host extra is also okay since the default value if set to false if nothing is passed, but this works too.